create_infotexts.py

```diff
@@ -333,5 +230,145 @@ def main():
+            names.append(flipped_name)
+        return names
+
+    def select_best_mapping_for_depicted_person(self, flipped_name):
+        """
+        Lookup available mappings for a string respresenting a name and return best choice.
+        
+        people_mapping is complete by design.
+        
+        The priority order is wikidata item < commons category < the name only.
+        :flipped_name: string representing full name turned around from
+            e.g. "surname, given name" -> "given name surname"
+            note that give_name is normally one word e.g. "Erfraim"
+        :return:  string representing the selcted mapping value
+        """
+        name_map = people_mapping[flipped_name]
+        name_as_wikitext = ""
+        if "wikidata" in name_map.keys():
+            name_as_wikitext = "[[:d:{wd_item}|{name}]]".format(
+                wd_item=name_map["wikidata"],
+                name=name_map["name"])
+
+        elif "commonscat" in name_map.keys():
+            name_as_wikitext = "[[Category:{commonscat}|{name}]]".format(
+                commonscat=name_map["commonscat"],
+                name=name_map["name"])
+
+        else:
+            name_as_wikitext = name_map["name"]
+
+        if "commonscat" in name_map.keys():
+            self.content_cats.append(name_map["commonscat"])
+
+        return name_as_wikitext
+
+
+    def process_depicted_place(self, place_string, item):
+        """
+        Create wikiformat depicted place string from raw input data.
+        
+        Populates data["depicted_place"] and interacts with data["commonscat"] and data["meta_cats"].
+        
+        :param place_string: string value <Ort, foto> in metadata item.  
+        :return: None (output stored in object attribute
+        """
+        places_mapping = load_places_mapping()
+        # print(places_mapping)
+
+        name_as_wikitext = ""
+
+        if place_string == "":
+            self.data["depicted_place"] = ""
+            return
+
+        elif place_string in places_mapping:
+            if not places_mapping[item["Ort, foto"]]["wikidata"] == "-":
+                self.data["depicted_place"] = "{{city|1={wikidata}|link=wikidata}}".format(
+                    wikidata=places_mapping[item["Ort, foto"]]["wikidata"][2:]
+                )
+            else:
+                self.data["depicted_place"] = "{{city|1={commons}|link=commons}}".format(
+                    commons=places_mapping[item["Ort, foto"]]["commons"]
+                )
+
+        elif place_string == "Stockholm": # Mainly interiors from buildings gardens
+            self.content_cats.append("[[Media_contributed_by_SMVK_2017-02_taken_in_Stockholm]]")
+            self.data["depicted_place"] = ""
+
+        elif place_string == "Macheras": # No WP article, highly ambiguous
+            self.data["depicted_place"] = ""
+        else:
+            raise ValueError("ValueError: Not able to map <Ort, foto>: {}".format(place_string))
```

Comment:
Don't raise a value you are not going to handle. The appropriate behaviour here would be to set a `meta_cat` and `place_as_wikitext = ""`.